src/coreclr/interpreter/compiler.cpp

```diff
@@ -3457,6 +3467,41 @@ int InterpCompiler::GenerateCode(CORINFO_METHOD_INFO* methodInfo)
                 m_ip += 1;
                 break;
 
+            case CEE_BOX: {
+                CHECK_STACK(1);
+                m_pStackPointer -= 1;
+                CORINFO_CLASS_HANDLE clsHnd = ResolveClassToken(getU4LittleEndian(m_ip + 1)),
+                    boxedClsHnd = m_compHnd->getTypeForBox(clsHnd);
```

Comment:
A nit - declaring two variables on a single line like this unusual for the coreclr codebase coding style.